src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.File.cs

```diff
@@ -29,6 +29,31 @@ public class Perf_File
         private Dictionary<int, string> _filesToRead;
         private string[] _linesToAppend;
         private Dictionary<int, string> _textToAppend;
+        private string _moveSource;
+        private string _moveDestination;
+
+        [IterationSetup(Target = nameof(Move))]
+        public void SetupMove()
+        {
+            _moveSource = FileUtils.GetTestFilePath();
+            _moveDestination = FileUtils.GetTestFilePath();
+
+            using var file = File.Create(_moveSource);
+            var data = ValuesGenerator.Array<byte>(OneMibibyte);
+            file.Write(data, 0, data.Length);
+        }
+
+        [Benchmark]
+        public void Move() => File.Move(_moveSource, _moveDestination);
+
+        [IterationCleanup(Target = nameof(Move))]
+        public void CleanupMove()
+        {
+            if (File.Exists(_moveDestination))
+            {
+                File.Delete(_moveDestination);
```

Comment:
I believe File.Delete does not need protecting with File.Exists

Reply:
I have taken the procedure from other methods in the class.